### ISSUE TITLE
Error from scheduler plugin during interpolation

### ISSUE THREAD

**Issue by landmanbester** [ratt-ru/QuartiCal#315]
**Describe the bug**
I'm getting the following error from the scheduler plugin during interpolation
```
# 2023-11-24 10:08:17 | INFO | interpolate:load_and_interpolate_gains | Beginning load/interpolate for B. Please be patient - this may take some time.
# 2023-11-24 10:08:19,965 - distributed.scheduler - ERROR - '<' not supported between instances of 'tuple' and 'str'
# Traceback (most recent call last):
#   File "/home/bester/.venv/pfb/lib/python3.10/site-packages/distributed/scheduler.py", line 4571, in _create_taskstate_from_graph
#     plugin.update_graph(
#   File "/home/bester/.venv/pfb/lib/python3.10/site-packages/quartical/scheduling/__init__.py", line 99, in update_graph
#     root_tokens = {tokenize(*sorted(v)): v for v in part_roots.values()}
#   File "/home/bester/.venv/pfb/lib/python3.10/site-packages/quartical/scheduling/__init__.py", line 99, in <dictcomp>
#     root_tokens = {tokenize(*sorted(v)): v for v in part_roots.values()}
# TypeError: '<' not supported between instances of 'tuple' and 'str'
# 2023-11-24 10:08:27 | SUCCESS | interpolate:load_and_interpolate_gains | Successfully loaded/interpolated B.
```
This does not seem to break anything fatally but it may indicate that something has changed in dask/distributed.

**Data description (if applicable)**
Data is available on bbking in /home/bester/projects/ESO137/msdir/ms1_primary.ms

**To Reproduce**
Seems to happen whenever I try to load and interpolate a gain with interpmode=ampphase and interpmethod=2dlinear using the distributed scheduler.  Here is a parset (converted to .txt so it will attach) that should reproduce the problem
[interp.txt](https://github.com/ratt-ru/QuartiCal/files/13456905/interp.txt)
 
**Version**
I am on the v0.2.1-dev branch using python3.10.8.
```
(qcal2) ╭─bester@bbking ~/projects/ESO137/qcal
╰─➤  pip freeze                                                                                                                                       130 ↵
aiobotocore==2.7.0
aiohttp==3.9.0
aioitertools==0.11.0
aiosignal==1.3.1
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
asciitree==0.3.3
astLib==0.11.10
astro-kittens==1.4.6
astro-tigger-lsm==1.7.2
astropy==5.3.4
async-timeout==4.0.3
attrs==23.1.0
bokeh==3.3.1
botocore==1.31.64
certifi==2023.11.17
charset-normalizer==3.3.2
cli-ui==0.17.2
click==8.1.7
cloudpickle==3.0.0
codex-africanus==0.3.4
colorama==0.4.6
Columnar==1.4.1
commonmark==0.9.1
configparser==6.0.0
contextlib2==21.6.0
contourpy==1.2.0
cycler==0.12.1
dask==2023.11.0
dask-ms==0.2.19
decorator==5.1.1
dill==0.3.7
distributed==2023.11.0
docopt==0.6.2
donfig==0.7.0
exceptiongroup==1.2.0
fasteners==0.19
fonttools==4.45.1
frozenlist==1.4.0
fsspec==2023.10.0
future==0.18.3
idna==3.4
importlib-metadata==6.8.0
iniconfig==2.0.0
Jinja2==3.1.2
jmespath==1.0.1
kiwisolver==1.4.5
llvmlite==0.41.1
locket==1.0.0
loguru==0.7.2
MarkupSafe==2.1.3
matplotlib==3.8.2
msgpack==1.0.7
multidict==6.0.4
munch==2.5.0
numba==0.58.1
numcodecs==0.12.1
numpy==1.26.2
omegaconf==2.3.0
packaging==23.2
pandas==2.1.3
partd==1.4.1
Pillow==10.1.0
pluggy==1.3.0
psutil==5.9.6
pydantic==1.10.13
pyerfa==2.0.1.1
Pygments==2.17.2
pyparsing==3.1.1
pytest==7.4.3
python-casacore==3.5.2
python-dateutil==2.8.2
pytz==2023.3.post1
PyYAML==6.0.1
-e git+ssh://git@github.com/ratt-ru/QuartiCal.git@c564aa815acf1ffb3cf3c7974a588bd9aec55e18#egg=quartical
requests==2.31.0
rich==12.6.0
ruamel.yaml==0.17.40
ruamel.yaml.clib==0.2.8
s3fs==2023.10.0
schema==0.7.5
scipy==1.11.4
six==1.16.0
sortedcontainers==2.4.0
stimela==2.0rc4
tabulate==0.8.10
tblib==3.0.0
tbump==6.11.0
tomli==2.0.1
tomlkit==0.11.8
toolz==0.12.0
tornado==6.3.3
typing_extensions==4.8.0
tzdata==2023.3
Unidecode==1.3.7
urllib3==2.0.7
wcwidth==0.2.12
wrapt==1.16.0
xarray==2023.11.0
xyzservices==2023.10.1
yarl==1.9.3
zarr==2.16.1
zict==3.0.0
zipp==3.17.0
```

**Comment by sjperkins** [comment]
There's been a change whereby collection keys are now represented as tuples rather than strings. This lines up with local collection keys,

**Comment by JSKenyon** [comment]
This should be fixed on the `v0.2.1-dev` branch. Closing for now, but please reopen if the problem persists.
